pyspeckit/spectrum/plotters.py

```diff
@@ -378,14 +402,13 @@ def label(self, title=None, xlabel=None, ylabel=None, verbose_label=False,
 
         if xlabel is not None:
             self.xlabel = xlabel
-        elif isinstance(self.Spectrum.xarr.xtype,str):
-            self.xlabel = self.Spectrum.xarr.xtype.title()
+        elif self._xunit:
+            self.xlabel += " ("+u.Unit(self._xunit).to_string()+")"
```

Comment:
This will only show `(km/s)`, for example.  We want something like "Velocity (km/s)".  This can be achieved with something like:
`"{0} ({1})".format(unit.physical_type.title(), unit.to_string())`
However, the physical type `speed` will show up where we want `Velocity`, so we need a lookuptable here.  Probably just a simple one:
`{'speed':'velocity'}` and the rest would just use their defaults.